pkg/sql/sqlbase/structured.proto

```diff
@@ -637,7 +637,7 @@ message TableDescriptor {
     // Nanoseconds since the Unix epoch.
     optional int64 expiration_time = 2 [(gogoproto.nullable) = false];
   }
-  optional SchemaChangeLease lease = 15;
+  optional SchemaChangeLease lease = 15 [deprecated = true];
```

Comment:
can be a followup, but I think we usually also rename the field to `DeprecatedFoo` since the annotation is only shown in some editors.